assets/src/blocks/Gallery/GalleryBlock.js

```diff
@@ -71,12 +88,17 @@ export const registerGalleryBlock = () => {
       }
     ],
     deprecated: [
+      {
```

Comment:
Deprecate the previous version. Remember that previous versions are adding chronologically, starting from newest to the oldest one.